marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/graph/searchNodes.sjs

```diff
@@ -177,8 +176,10 @@ result.map(item => {
     edges.push(edge);
     if (!nodes[objectId]) {
       let objectNode = {};
-      objectNode.id = item.firstObjectIRI;
-      objectNode.docUri = objectUri;
+      objectNode.id = objectId;
+      if (item.nodeCount === 1) {
```

Comment:
docURI is being set again in line 185. Is the null not setting correctly that we need to add this?

Reply:
With my testing by manually copying the nodes/edges into a VisJS JSfiddle, the edge wasn't connecting to the group node. This change fixed that.

Reply:
Ok. Please remove 187. It would be undoing or re-assigning this.

Reply:
Ah. I thinking of the change on 179 that made the graph edge work. Good catch on the duplicate assignment. I've updated that now.